lib/matplotlib/tests/test_figure.py

```diff
@@ -232,7 +232,7 @@ def test_add_subplot_invalid():
 def test_suptitle():
     fig, _ = plt.subplots()
     fig.suptitle('hello', color='r')
-    fig.suptitle('title', color='g', rotation='30')
+    fig.suptitle('title', color='g', rotation=30)
```

Comment:
I guess strictly speaking this is backcompat breaking?  I doubt it's worth a deprecation period, but it should be noted.

Reply:
Not even sure we need to mention it. '30' is not a supported format [according to docs](https://matplotlib.org/stable/api/text_api.html#matplotlib.text.Text.set_rotation). It just happens to work. If we drop that without deprecation period, a note does not really help anybody. Nobody goes through the note and says "hey rotation does not take string numbers anymore. That's my thing, I'm doing that all the time". I'll bet anything if anybody passes '30', they will only notice by the error message they will get.